OSDOCS-12145: Adds snippet about unified perspective (#93186)
openshift/openshift-docs: snippets/unified-perspective-web-console.adoc

// Snippet included in the following assemblies and modules:
//
// * getting-started/openshift-overview.adoc
// * web_console/web-console-overview.adoc
// * modules/about-administrator_web-console.adoc
// * modules/about-developer_web-console.adoc
// * modules/viewing-stats-collected-kubernetes-nmstate-op.adoc
// * modules/viewing-a-project-using-the-web-console.adoc
// * modules/red-hat-marketplace-features.adoc

:_mod-docs-content-type: SNIPPET

[IMPORTANT]
====

> Starting with {product-title} 4.19, the perspectives in the web console have unified. There is no longer a *Developer* perspective by default.

Reviewer: ```suggestion
Starting with {product-title} 4.19, the perspectives in the web console are unified. The *Developer* perspective is no longer enabled by default.
```

> All {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission to certain features from the cluster owner.

Reviewer: ```suggestion
All users can interact with all {product-title} web console features. However, if you are not the cluster owner, you might need to request permission to certain features from the cluster owner.
```

> You can still enable the *Developer* perspective. The *Getting Started* pane in the web console provides resources such as, a tour of the console, information on setting up your cluster, a quick start for enabling the *Developer* perspective, and links to explore new features and capabilities.

Reviewer: ```suggestion
You can still enable the *Developer* perspective. On the *Getting Started* pane in the web console, you can  take a tour of the console, find information on setting up your cluster, view a quick start for enabling the *Developer* perspective, and follow links to explore new features and capabilities.
```

// Snippet included in the following assemblies and modules:
//
// * getting-started/openshift-overview.adoc
// * web_console/web-console-overview.adoc
// * modules/about-administrator_web-console.adoc
// * modules/about-developer_web-console.adoc
// * modules/viewing-stats-collected-kubernetes-nmstate-op.adoc
// * modules/viewing-a-project-using-the-web-console.adoc
// * modules/red-hat-marketplace-features.adoc

:_mod-docs-content-type: SNIPPET

[IMPORTANT]
====
Starting with {product-title} 4.19, the perspectives in the web console have unified. There is no longer a *Developer* perspective by default.

All {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission to certain features from the cluster owner.

You can still enable the *Developer* perspective.

> See xref:../web_console/web-console-overview.adoc#enabling-developer-perspective_web-console_web-console-overview for more information on enabling the *Developer* perspective.

Reviewer 2: Unless you've gotten an exception for this, xrefs are not allowed in snippets unfortunately: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets
